x-pack/plugins/cross_cluster_replication/public/app/components/follower_index_resume_provider.js

```diff
@@ -18,7 +19,7 @@ import routing from '../services/routing';
 import { resumeFollowerIndex } from '../store/actions';
 import { arrify } from '../../../common/services/utils';
 
-class Provider extends PureComponent {
+class FollowerIndexResumeProviderUi extends PureComponent {
```

Comment:
We used `Ui` (or `Component`) postfix to differ the initial component and the wrapper: `ComponentName = injectI18n(ComponentNameUi)`.
Since we are getting rid of `injectI18n`, we can remove `Ui` at the end of class name. 
But it's up to you, you can leave it as well.